scripts/minimal_init.lua

```diff
@@ -13,4 +13,8 @@ vim.api.nvim_exec([[
   runtime! plugin/plenary.vim
 ]], false)
 
-require('lean').setup{}
+require('lean').setup{
+  abbreviations = {
+    snippets = true,
```

Comment:
It's a bit annoying that I still can't figure out a good way to avoid all the global state involved here, otherwise we'd be able to test the different implementations. But the neovim testing scene is still super immature, so oh well I suppose.